tests/unit/ps-dockerfile.bats

```diff
@@ -10,44 +10,36 @@ teardown() {
   destroy_app
 }
 
-@test "ps (dockerfile)" {
-  # CI support: 'Ah. I just spoke with our Docker expert --
-  # looks like docker exec is built to work with docker-under-libcontainer,
-  # but we're using docker-under-lxc. I don't have an estimated time for the fix, sorry
-  skip "circleci does not support docker exec at the moment."
-  run bash -c "dokku ps $TEST_APP | grep -q \"node web.js\""
-  echo "output: "$output
-  echo "status: "$status
-  assert_success
-}
+# @test "ps (dockerfile)" {
```

Comment:
This is commented out similar to the other `dokku ps` test since circleCI doesn't support `docker exec`. Note that it is also "skipped" with the bats `skip` command. Additionally bats will run the `setup` and `teardown` functions even on skipped tests. Therefore commenting this out speeds up the test suite by not superfluously running these functions for test that will not actually be executed. I have an issue opened with the bats project for this as well.